public/index.html

```
@@ -0,0 +1,664 @@
+</head>
+<body>
+
+<!-- HEADER -->
+<header class="header">
+  <div class="logo">Career Ops</div>
+  <div class="header-divider"></div>
+  <div class="header-stats" id="header-stats">
+    <span class="header-stat" style="color:var(--muted)">Loading...</span>
+  </div>
+  <div class="header-actions">
+    <button class="btn" onclick="loadData()">&#8635; Refresh</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)